tests/sqllogictests/suites/mode/standalone/ee/explain_agg_index.test

```diff
@@ -325,7 +325,7 @@ EXPLAIN SELECT station_name, MIN(measurement) AS min_measurement, AVG(measuremen
 ----
 EvalScalar
 ├── output columns: [MIN(measurement) (#2), MAX(measurement) (#5), onebrc.station_name (#0), mean_measurement (#6)]
-├── expressions: [sum(measurement) (#3) / CAST(if(CAST(count(measurement) (#4) = 0 AS Boolean NULL), 1, count(measurement) (#4)) AS UInt64 NULL)]
+├── expressions: [sum(measurement) (#3) / CAST(if(CAST(count(measurement) (#4) = 0 AS Boolean NULL), 1, count(measurement) (#4)) AS Float64 NULL)]
```

Comment:
In this pr, we remove int/double arithmetic, so the type changed. I think it's ok.

And after we split crate, we also can contain int/double arithmetic.

Reply:
> And after we split crate, we also can contain int/double arithmetic.

Yes, Let's just split the crate and don't remove registers which keep the behavior as before.

If something incompat happens, we don't need to revert this pr. 

We can remove these after this pr.

Reply:
Ok, I keep the registers function. https://github.com/databendlabs/databend/pull/17133/commits/4d189c240b023388ff368ce3ed460dfb463d08a8

![image](https://github.com/user-attachments/assets/9bedbbb5-ad3e-4dcc-9cbe-f84541412692)